var/spack/repos/builtin/packages/vtk-m/mr3259-thrust-is_arithmetic-fix.patch

```diff
@@ -0,0 +1,35 @@
+diff --git a/vtkm/exec/cuda/internal/WrappedOperators.h b/vtkm/exec/cuda/internal/WrappedOperators.h
+index bece57098..b46077e6a 100644
+--- a/vtkm/exec/cuda/internal/WrappedOperators.h
++++ b/vtkm/exec/cuda/internal/WrappedOperators.h
+@@ -22,6 +22,10 @@ VTKM_THIRDPARTY_PRE_INCLUDE
+ #include <thrust/system/cuda/memory.h>
+ VTKM_THIRDPARTY_POST_INCLUDE
+ 
++#if THRUST_VERSION >= 200500
++#include <cuda/std/type_traits>
++#endif
++
+ namespace vtkm
+ {
+ namespace exec
+@@ -193,11 +197,19 @@ namespace detail
```

Comment:
we will probably have to update this numbers

Reply:
Thanks @pauleonix for updating this, we can bring it upstream. This is good for the spack package as long as it applies

Reply:
I took the fix from upstream :laughing:

Reply:
The patch comes from @kmorel (this [PR](https://gitlab.kitware.com/vtk/vtk-m/-/merge_requests/3259)) and seems to apply cleanly to 2.0.0, 2.1.0 and 2.2.0 according to my testing. Can I resolve this conversation?